<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProbeConfig</class>
 <widget class="QWidget" name="ProbeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="3">
      <widget class="QGroupBox" name="sphereGroupBox">
       <property name="title">
        <string>Sphere</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="sphereRadius">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sphereCenterXSpinBox"/>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="sphereCenterYSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Center</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Radius</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="sphereCenterZSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="sourceCombo">
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sphere</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Box</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Implicit Plane</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open ...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QGroupBox" name="lineGroupBox">
       <property name="title">
        <string>Line</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="linePoint2XSpinBox"/>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="linePoint1XSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Point1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Point2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="linePoint1YSpinBox"/>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="linePoint2YSpinBox"/>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="linePoint1ZSpinBox"/>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="linePoint2ZSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QGroupBox" name="implicitPlaneGroupBox">
       <property name="title">
        <string>Implicit Plane</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="normalZSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Normal</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="originXSpinBox"/>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="originZSpinBox"/>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="originYSpinBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="normalXSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Origin</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="normalYSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Plot</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="plotLayout"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
